src/components/ChatEntry.js

```diff
@@ -1,22 +1,39 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const likeMe = () => {
+    props.likeCallback(props.id);
+  };
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div
+      className={
+        props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote'
+      }
```

Comment:
Tip: Hardcoding Vladamir is a little brittle (if we changed the data in the JSON file it would break your code) but works in this case.